tests/yam/agama/validate_product_installed_agama.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that the product installed by agama via /etc/os-release
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use Config::Tiny;
+use Test::Assert ':all';
+use scheduler 'get_test_suite_data';
+
+sub run {
+    select_console 'root-console';
+
+    my ($product) = @{get_test_suite_data()->{test_data}}{product};
```

Comment:
when you use the right path (as suggested above) you don't need to access `test_data` hash.